dev-scripts.sh

```
@@ -0,0 +1,353 @@
+    fi
+    
+    npm test
+    if [ $? -ne 0 ]; then
+        print_error "Types package tests failed"
+        return 1
+    fi
+
+    # Test backend integration
+    if [ -d "$tlbot_path/backend" ]; then
+        print_step "Testing backend integration..."
+        cd "$tlbot_path/backend"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)